.gitignore

```diff
@@ -86,3 +86,6 @@ packages/**/docs/prose/build/
 packages/**/docs/api/build/
 packages/**/docs/api/source/api/
 packages/**/docs/dist/
+
+# Rollup-built code
+umd/
```

Comment:
Shouldn't `packages/**/dist` and `packages/**/umd` also need to be in here?

Reply:
As I understand it correctly, `umd/` matches the directories that `packages/**/umd` would match, but it's more generic. 
> frotz/ matches frotz and a/frotz that is a directory
from https://git-scm.com/docs/gitignore.

I don't have a strong opinion on this, but I can change it if that makes it clearer. And `dist/` is already present upper in the file: if I change to `packages/**/umd`, i'll consistently change to `packages/**/dist` as well.

Reply:
Nope you're right, I don't think there will be any `umd` folder anywhere that we _do_ want to include, so just the op-level definition is fine then if that works.